common.blocks/menu/menu.js

```diff
@@ -134,8 +134,10 @@ provide(BEMDOM.decl({ block : this.name, baseBlock : Control }, /** @lends menu.
     _onItemHover : function(item) {
         if(item.hasMod('hovered')) {
             this._hoveredItem && this._hoveredItem.delMod('hovered');
+            this.domElem.attr('aria-activedescendant', item.domElem.attr('id'));
```

Comment:
What if we add `MenuItem#getId()` method, which will return cached `id` property. 

~~~js
// menu-item.js
onSetMod : {
  'js' : {
    'inited' : function() {
      this._id = this.domElem.attr('id');
    }
  }
},

getId : function() {
  return this._id;
}
~~~

Does this DOM attribute getters efficient, in case we check them in every mouseover? 

//cc @dfilatov @veged